assets/charts/control-plane/kube-apiserver/Chart.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: v1
+apiVersion: v2
```

Comment:
Why is this line duplicated? Here and https://github.com/kinvolk/lokomotive/pull/1030/files#diff-8d204efa836ea1ad446bacaf8bd7e9aca251258fc0b499a536cd5a1e53525c1cR18 ?

Isn't it overriden? (sorry, helm newbie, but I guess yaml parser overrides this before helm really use it?)

Reply:
Oh they are different. This is apiVersion and the other is app version.

Reply:
:joy: Sorry for the noise, then! I'll have a break :D